app/webpacker/controllers/order_info_controller.js

```diff
@@ -3,21 +3,16 @@ import { Controller } from "stimulus"
 export default class extends Controller {
   static values = { resource: String, shipmentNumber: String }
 
-  connect() {
-    console.log('connected')
-    console.log(this.shipmentNumberValue)
```

Comment:
yeah sorry, I had some `console.logs` in my earlier commits so that I could verify somethings were working properly! Removed them all in my last commit though but I should've not included them in those commits in the first place!

Reply:
You can squash them, and then have a clean git history 💪